files/es/web/api/notification/dir/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Notification.dir
+slug: Web/API/notification/dir
+tags:
+  - API
+  - Notification
+  - Notifications
+  - Notifications API
+  - Property
+  - Reference
+  - dir
+translation_of: Web/API/Notification/dir
+browser-compat: api.Notification.dir
+---
+{{APIRef("Web Notifications")}}{{AvailableInWorkers}}{{SeeCompatTable()}}
+
+La propiedad `dir` de solo lectura de la interfaz {{domxref("Notification")}} indica la direccion del texto en la notificación, como se especifica en la opción `dir` del constructor {{domxref("Notification.Notification","Notification()")}}.
+
+## Sintaxis
+
+```js
+var direction = Notification.dir;
+```
+
+### Valor
+
+Un {{domxref("DOMString")}} especifica la dirección del texto, los posibles valores pueden ser:
+
+- `auto`: Aplica el comportamiento del idioma del navegador (por defecto).
+- `ltr`: El texto es desplegado de izquierda a derecha.
+- `rtl`: El texto es desplegado de derecha a izquierda.
+
+> **Nota:** La mayoría de los navegadores parecen ignorar las intrucciones explicitas ltr y rtl, y solo van a las proveídas por el navegador.
```

Comment:
This isn't in the en-US, but I personally think using `code` formatting makes it more readable/clear

```suggestion
> **Nota:** La mayoría de los navegadores parecen ignorar las intrucciones explicitas `ltr` y `rtl`, y solo van a las proveídas por el navegador.
```

Reply:
I'm agree, let's apply this change, thank you!